src/material-examples/sidenav-fab/sidenav-fab-example.css

```diff
@@ -14,10 +14,6 @@
   overflow: visible;
 }
 
-.example-scrolling-content {
```

Comment:
This is actually needed because the same class is used to make the sidenav content scroll. Actually can you add `height: 100%` to this instead? That should fix the issue where the FAB button in the content section scrolls up instead of staying fixed:

```css
.example-scrolling-content {
  overflow: auto;
  height: 100%;
}
```

Reply:
Thank you , I fixed it in the next commit